build.gradle

```diff
@@ -0,0 +1,93 @@
+repositories {
+    mavenCentral()
+}
+
+configurations {
+    toCopy
+}
+
+dependencies {
+    toCopy 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.47'
+}
+
+task downloadSwaggerCodegen(type: Copy) {
+    from configurations.toCopy
+    into 'lib'
+    rename {
+        String fileName -> "swagger-codegen-cli.jar"
+    }
+}
+
+task createLanguageModule() {
```

Comment:
We're going to just have this be in the readme, we don't need tooling for this and it won't happen often.